internal/business/persisted_operations/loader.go

```diff
@@ -31,7 +32,7 @@ func determineLoader(cfg Config, log *slog.Logger) (RemoteLoader, error) {
 	if cfg.Remote.GcpBucket != "" {
 		loader, err := NewGcpStorageLoader(context.Background(), cfg.Remote.GcpBucket, cfg.Store, log)
 		if err != nil {
-			return nil, errors.New("unable to instantiate GcpBucketLoader")
+			return nil, errors.New(fmt.Sprintf("unable to instantiate GcpBucketLoader err: %s", err))
```

Comment:
return nil, fmt.Errorf("unable to instantiate GcpBucketLoader %w", err)
https://pkg.go.dev/errors